packages/opencanary/data_stream/events/elasticsearch/ingest_pipeline/default.yml

```
@@ -48,6 +48,28 @@ processors:
       target_field: message
       ignore_missing: true
       ignore_failure: true
+  - set:
+      description: Set event.category according to event details
+      tag: set_event_category
+      field: event.category
+      value:
+        - network
+        - intrusion_detection
+      if: ctx.opencanary?.logtype != null && ![1000, 1001, 1002, 1003, 1004, 1005, 1006].contains(ctx.opencanary.logtype)
```

Since opencanary.logtype is a long, can we
```suggestion
      if: ctx.opencanary?.logtype != null && (ctx.opencanary.logtype < 1000 || 1006 < ctx.opencanary.logtype)
```
instead? This saves an allocation, and reduces the number of comparisons. Similar below.